recipes/tibanna/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "tibanna" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "aa63462bb2e114442baca9cb1ee25980c3f03356d2c280ed7eabf694c79061db"
+
+build:
+  number: 0
+  entry_points:
+    - tibanna = tibanna.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [not py36]
+
+requirements:
+  host:
+    - benchmark-4dn ==0.5.6
+    - boto3 >=1.9.0
+    - botocore >=1.12.1
+    - pip
+    - python
+    - python-lambda-4dn ==0.12.2
+    - requests ==2.22.0
+    - urllib3 >=1.24
+  run:
+    - benchmark-4dn ==0.5.6
+    - boto3 >=1.9.0
+    - botocore >=1.12.1
+    - python
+    - python-lambda-4dn ==0.12.2
+    - requests ==2.22.0
+    - urllib3 >=1.24
```

It seems you do need ``futures``
```suggestion
    - urllib3 >=1.24
    - futures
```

---

Are you going to build this package for Python 2.7 or just 3.7+ ?

---

Tibanna does only support py3k, so we don't need the py27 build. However, there is no futures in py3k...